internal/gke/kubernetes.go

```diff
@@ -138,6 +140,65 @@ func (c *kubernetesClient) GetFetchableResourceTypes() ([]*ResourceType, error)
 	return resourceTypes, nil
 }
 
+func (c *kubernetesClient) GetResources(toBeFetched []*ResourceType, namespaces []string) ([]*Resource, error) {
+	var resources []*Resource
+
+	maxGoroutines := 10
```

Comment:
it would be nice to define sth like `defaultMaxGoroutines` as const
and also a param to the `kubernetesClient`, init it with default on creation